src/tasks/tasks.hpp

```diff
@@ -106,7 +113,11 @@ class Task {
   template <typename TID>
   Task(TID &&dep, const std::string &label, const std::function<TaskStatus()> &func,
        std::pair<int, int> limits = {1, 1})
-      : label_(label), f(func), exec_limits(limits) {
+      : Task(std::forward<TID>(dep), label, 0, func, limits) {}
+  template <typename TID>
+  Task(TID &&dep, const std::string &label, int verbose_level,
+       const std::function<TaskStatus()> &func, std::pair<int, int> limits = {1, 1})
+      : label_(label), verbose_level_(verbose_level), f(func), exec_limits(limits) {
```

Comment:
Add verbosity flag to tasks, if `verbose_level_ > 0` the task will print its name out to stdout before starting the task. Useful for debugging.

Reply:
This is a useful feature.  I wonder if the verbosity should be blanket applied to all tasks based off a `verbose` flag set in `pin`, rather than enrolled via each `AddTask`?

Reply:
I agree that would be nice, but I think that it requires a bit of threading to make it work (since the tasking stuff doesn't directly access `pin` currently). I would leave this for a future PR. 

In practice, I just manually comment out the `verbose_level_ > 0` test and recompile if I want to see everything...